modules/client/src/main/java/org/jboss/wsf/stack/cxf/client/serviceref/CXFHandlerResolverImpl.java

```diff
@@ -328,7 +331,8 @@ private void processHandlerElement(Element el, @SuppressWarnings("rawtypes") Lis
               .unmarshal(el, PortComponentHandlerType.class).getValue();
           chain.addAll(buildHandlerChain(pt, classLoader));
       } catch (JAXBException e) {
-          e.printStackTrace();
+         DEPLOYMENT_LOGGER.unableToProcessHandlerElement(el, e);
+         log.error(e);
```

Comment:
Isn't the whole cause exception logged at the line above? Do we really need log invocations?